sphinx_markdown/directives.py

```diff
@@ -1,7 +1,10 @@
 
 from docutils.parsers.rst import directives
 from sphinx.ext.autodoc import Options
-from sphinx.util.compat import Directive
+try:
+    from sphinx.util.compat import Directive
+except ImportError:
+    from docutils.parsers.rst import Directive
```

Comment:
You can unconditionally use this one as the base.
In earlier versions of sphinx, they exported Directive as part of their interface, but it was still just docutils.parsers.rst.Directive. Then they deprecated it, and added a small wrapper to say it was deprecated, and finally removed it from the interface.